DEVDOC.md

```diff
@@ -1,5 +1,43 @@
 # GAPID Developer Documentation
 
+## Setup Golang development
+
+This project contains Golang code, but it does not have the file hierarchy of
+regular Golang projects (this is due to the use of Bazel as a build system).
+The `cmd/gofuse` utility enables to re-create the file hierarchy expected by Go
+tools:
+
+```sh
+# We assume GAPID has been checked out under ~/gapid
+cd ${HOME}/gapid
```

Comment:
I propose "cd <path-to-gapid-source>", which is independent of the OS, doesn't need the comment and is consistent with BUILDING.md

Reply:
omg, github cuts my stuff.
I wrote '''cd <path-to-gapid-source>'''

Reply:
arg, again
```cd <path-to-gapid-source>```

Reply:
done